gateway/Cargo.toml

```diff
@@ -10,13 +10,13 @@ edition = "2018"
 pancurses = "0.16.1"
 rand = "0.8.4"
 rdkafka = { version = "0.25", features = ["cmake-build"] }
-env_logger = "0.8.4"
+env_logger = "0.9.0"
 
 base64 = "0.13"
 clap = "3.0.0-beta.5"
 tokio = { version = "1.0", features = ["macros", "rt-multi-thread", "time"] }
-futures = "0.3.0"
-libc = "0.2.0"
+futures = "0.3.18"
+libc = "0.2.108"
```

Comment:
`Cargo.toml` shouldn't be modified here

Reply:
Reverted these changes in a new commit

Reply:
@subhommitra please make sure that `Cargo.lock` is also unaffected. 

Maybe, it's better to rebase to `main` branch (or merge `main` into this branch) and accept changes from `main` for files under `gateway/`

Reply:
Merged `main` into this branch and pushed. Please review.

Reply:
LGTM!